vmm/src/vm_config.rs

```diff
@@ -314,6 +318,32 @@ pub fn default_netconfig_queue_size() -> u16 {
     DEFAULT_NET_QUEUE_SIZE
 }
 
+fn serialize_netconfig_fds<S>(x: &Option<Vec<i32>>, s: S) -> Result<S::Ok, S::Error>
+where
+    S: serde::Serializer,
+{
+    if let Some(x) = x {
+        warn!("'NetConfig' contains FDs that can't be serialized correctly. Explicitly set them as invalid.");
```

Comment:
The phrase "Explicitly set them as invalid." sounds like an instruction to the user. I think we can drop the warnings completely - there isn't anything useful that the user can do about this.

Reply:
I was hoping to use this message to warn users they are migrating/restoring VMs using netfds (which won't be serialized correctly) and they should act accordingly, say provide the valid `netfds` from the destination/restoring VM side.

Maybe a better message would be: `'NetConfig' contains FDs that can't be serialized correctly. Serializing them as invalid FDs."`?

Reply:
Sounds good to me.